Sources/System/Sockets/SocketOperations.swift

```diff
@@ -0,0 +1,253 @@
+/*
+ This source file is part of the Swift System open source project
+
+ Copyright (c) 2021 Apple Inc. and the Swift System project authors
+ Licensed under Apache License v2.0 with Runtime Library Exception
+
+ See https://swift.org/LICENSE.txt for license information
+*/
+
+extension SocketDescriptor {
+  /// Create an endpoint for communication.
+  ///
+  /// - Parameters:
+  ///    - domain: Select the protocol family which should be used for
+  ///      communication
+  ///    - type: Specify the semantics of communication
+  ///    - protocol: Specify a particular protocol to use with the socket.
+  ///      Normally, there is only one protocol for a particular connection
+  ///      type within a protocol family, so a default argument of `.default`
+  ///      is provided
```

Comment:
Formatting is off by one I think:

```suggestion
  ///   - domain: Select the protocol family which should be used for
  ///     communication
  ///   - type: Specify the semantics of communication
  ///   - protocol: Specify a particular protocol to use with the socket.
  ///     Normally, there is only one protocol for a particular connection
  ///     type within a protocol family, so a default argument of `.default`
  ///     is provided
```